com.unity.uiwidgets/Runtime/gestures/multitap.cs

```diff
@@ -228,6 +258,16 @@ public DoubleTapGestureRecognizer(object debugOwner = null, PointerDeviceKind? k
             }
         }
 
+        void _checkUp(_TapTracker tracker) {
+            // D.assert(buttons == kPrimaryButton);
+            if (onDoubleTap != null) {
+                invokeCallback<object>("onDoubleTap", () => {
+                    onDoubleTap(new DoubleTapDetails(tracker._initialGlobalPosition));
```

Comment:
the delegate GestureDoubleTapCallback should take zero argument, is it a typo here?

Reply:
fixed